includes/extra_configures/enable_error_logging.php

```diff
@@ -81,7 +81,7 @@ function zen_debug_error_handler($errno, $errstr, $errfile, $errline)
     if (!empty($backtrace)) {
         $backtrace = PHP_EOL . rtrim($backtrace);
     }
-    $language_info = ', Language id ' . $_SESSION['languages_id'];
+    $language_info = ', Language id ' . ($_SESSION['languages_id'] ?? 'SESSION[languages_id] NOT DEFINED'); 
```

Comment:
How about a simple 'not set', like the IP address?

Reply:
At the time of entry, it seemed more informative to have something other than "not set" which can occur in a fully executed and completed load as compared to a call occurring before the value is available.

Modified it anyways to align with the other "not set" type values.